tests/attributes_iter.rs

```diff
@@ -0,0 +1,25 @@
+mod test_data;
+use cloudevents::event::AttributeValue;
+use test_data::*;
+
+#[test]
+fn iter_v10_test() {
+    let in_event = v10::full_no_data();
+    let mut iter_v10 = in_event.attributes_iter();
+
+    assert_eq!(
+        ("id", AttributeValue::String("0001")),
+        iter_v10.next().unwrap()
```

Comment:
uh wait, should the first attribute be `specversion`?

Reply:
done